### PR TITLE
feat!: Switch to using sls_sdk import

### PR DESCRIPTION
**Pull request by Mmarzex**
To fix Dashboard SDK collision

### CODE REVIEW

File: python/packages/aws-lambda-sdk/serverless_aws_lambda_sdk/instrument/__init__.py [hunk 1]
```diff
@@ -8,15 +8,15 @@
...
 from .lib.sdk import serverlessSdk
 from .lib.invocation_context import (
     set as set_invocation_context,
     get as get_invocation_context,
 )
 from .lib.payload_conversion import to_trace_payload, to_request_response_payload
-from serverless_sdk.lib.trace import TraceSpan
-from serverless_sdk.lib.captured_event import CapturedEvent
+from sls_sdk..lib.trace import TraceSpan
+from sls_sdk..lib.captured_event import CapturedEvent
```
Comment (Danwakeem) [inline python/packages/aws-lambda-sdk/serverless_aws_lambda_sdk/instrument/__init__.py]:
```suggestion
from sls_sdk.lib.trace import TraceSpan
from sls_sdk.lib.captured_event import CapturedEvent
```

File: python/packages/aws-lambda-sdk/serverless_aws_lambda_sdk/instrument/__init__.py [hunk 2]
```diff
@@ -8,15 +8,15 @@
 from typing import List, Optional, Any
 from typing_extensions import Final
 import random
-from serverless_sdk.lib.timing import to_protobuf_epoch_timestamp
+from sls_sdk..lib.timing import to_protobuf_epoch_timestamp
```
Comment (Danwakeem) [inline python/packages/aws-lambda-sdk/serverless_aws_lambda_sdk/instrument/__init__.py]:
```suggestion
from sls_sdk.lib.timing import to_protobuf_epoch_timestamp
```

File: python/packages/aws-lambda-sdk/serverless_aws_lambda_sdk/__init__.py [hunk 3]
```diff
@@ -2,10 +2,10 @@
 import os
 import logging
 from typing import Optional
-from serverless_sdk import serverlessSdk as baseSdk
+from sls_sdk. import serverlessSdk as baseSdk
```
Comment (Danwakeem) [inline python/packages/aws-lambda-sdk/serverless_aws_lambda_sdk/__init__.py]:
```suggestion
from sls_sdk import serverlessSdk as baseSdk
```

File: python/packages/aws-lambda-sdk/serverless_aws_lambda_sdk/__init__.py [hunk 4]
```diff
@@ -2,10 +2,10 @@
...
 from .trace_spans.aws_lambda import aws_lambda_span
-from serverless_sdk import ServerlessSdk, TraceSpans
-from serverless_sdk.lib.trace import TraceSpan
+from sls_sdk. import ServerlessSdk, TraceSpans
```
Comment (Danwakeem) [inline python/packages/aws-lambda-sdk/serverless_aws_lambda_sdk/__init__.py]:
```suggestion
from sls_sdk import ServerlessSdk, TraceSpans
```

File: python/packages/aws-lambda-sdk/serverless_aws_lambda_sdk/__init__.py [hunk 5]
```diff
@@ -2,10 +2,10 @@
...
+from sls_sdk..lib.trace import TraceSpan
```
Comment (Danwakeem) [inline python/packages/aws-lambda-sdk/serverless_aws_lambda_sdk/__init__.py]:
```suggestion
from sls_sdk.lib.trace import TraceSpan
```

File: python/packages/aws-lambda-sdk/serverless_aws_lambda_sdk/internal_extension/wrapper.py [hunk 6]
```diff
@@ -15,7 +15,7 @@
 from ..instrument import Instrumenter
 
 if TYPE_CHECKING:
-    from serverless_sdk.sdk.base import ServerlessSdk
+    from sls_sdk..sdk.base import ServerlessSdk
```
Comment (Danwakeem) [inline python/packages/aws-lambda-sdk/serverless_aws_lambda_sdk/internal_extension/wrapper.py]:
```suggestion
    from sls_sdk.sdk.base import ServerlessSdk
```

File: python/packages/aws-lambda-sdk/serverless_aws_lambda_sdk/trace_spans/aws_lambda.py [hunk 7]
```diff
@@ -3,7 +3,7 @@
 import platform
 import logging
 
-from serverless_sdk.lib.trace import TraceSpan
+from sls_sdk..lib.trace import TraceSpan
```
Comment (Danwakeem) [inline python/packages/aws-lambda-sdk/serverless_aws_lambda_sdk/trace_spans/aws_lambda.py]:
```suggestion
from sls_sdk.lib.trace import TraceSpan
```

File: python/packages/aws-lambda-sdk/tests/fixtures/lambdas/sdk.py [hunk 8]
```diff
@@ -1,4 +1,4 @@
-from serverless_sdk import serverlessSdk as sdk
+from sls_sdk. import serverlessSdk as sdk
```
Comment (Danwakeem) [inline python/packages/aws-lambda-sdk/tests/fixtures/lambdas/sdk.py]:
```suggestion
from sls_sdk import serverlessSdk as sdk
```

File: python/packages/aws-lambda-sdk/tests/fixtures/lambdas/sdk_sampled_out.py [hunk 9]
```diff
@@ -7,7 +7,7 @@ def handler(event, context):
     counter += 1
     invocation_id = counter
 
-    from serverless_sdk import serverlessSdk as sdk
+    from sls_sdk. import serverlessSdk as sdk
```
Comment (Danwakeem) [inline python/packages/aws-lambda-sdk/tests/fixtures/lambdas/sdk_sampled_out.py]:
```suggestion
    from sls_sdk import serverlessSdk as sdk
```

File: python/packages/aws-lambda-sdk/tests/instrument/lib/test_dev_mode.py [hunk 10]
```diff
@@ -1,7 +1,7 @@
 from unittest.mock import MagicMock, call
-from serverless_sdk.lib.captured_event import CapturedEvent
-from serverless_sdk.lib.trace import TraceSpan
-from serverless_sdk_schema import TracePayload
+from sls_sdk..lib.captured_event import CapturedEvent
+from sls_sdk..lib.trace import TraceSpan
```
Comment (Danwakeem) [inline python/packages/aws-lambda-sdk/tests/instrument/lib/test_dev_mode.py]:
```suggestion
from sls_sdk.lib.captured_event import CapturedEvent
from sls_sdk.lib.trace import TraceSpan
```

**Review by Danwakeem — APPROVED** [review]
:shipit: Sorry python gang 😅

File: python/packages/aws-lambda-sdk/CHANGELOG.md [hunk 11]
```diff
@@ -2,6 +2,11 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## 0.1.11 (2023-03-30)
+### ⚠ BREAKING CHANGES
```
Comment (medikoo) [inline python/packages/aws-lambda-sdk/CHANGELOG.md:6]: @Mmarzex Breaking changes should have a minor bump (technically it should have been v0.2 release)

**Pull request merged**